dis_snek/models/discord/color.py

```diff
@@ -60,19 +60,19 @@ def _get_byte(self, n) -> int:
 
     @property
     def r(self) -> int:
-        return self._get_byte(0)
+        return self._get_byte(2)
 
     @property
     def g(self) -> int:
         return self._get_byte(1)
 
     @property
     def b(self) -> int:
-        return self._get_byte(2)
+        return self._get_byte(0)
 
     @property
     def rgb(self) -> Tuple[int, int, int]:
-        return self._get_byte(0), self._get_byte(1), self._get_byte(2)
+        return self._get_byte(2), self._get_byte(1), self._get_byte(0)
```

Comment:
actually looking at this, why didn't we just do `return self.r, self.g, self.b`

Reply:
Good point, made that change